regolith/builders/recentcollabsbuilder.py

```diff
@@ -139,15 +151,18 @@ def get_ppl_inst_info(self, id, months):
                 emp.sort(key=ene_date_key, reverse=True)
         return ppl_names
 
-    def make_csv_and_excel(self, id, months):
-        '''
+    def make_csv_and_excel(self, ppl_names):
```

Comment:
Perhaps ``ppl`` is better than ``ppl_names`` since it is a list of tuples that contain more than the names